src/instrumentsscene/view/layoutpaneltreemodel.cpp

```diff
@@ -69,6 +69,7 @@ LayoutPanelTreeModel::LayoutPanelTreeModel(QObject* parent)
         updateRearrangementAvailability();
         updateRemovingAvailability();
         updateSelectedItemsType();
+        emit isAddingSystemMarkingsAvailableChanged(isAddingSystemMarkingsAvailable());
```

Comment:
We can do it more optimally (for better performance and to match the current style):

1. add m_isAddingSystemMarkingsAvailable and updateIsAddingSystemMarkingsAvailable. It will update the flag accordingly and emit the signal if need
2. call update here and wherever needed